packages/web/src/brandkit/Color.tsx

```diff
@@ -0,0 +1,150 @@
+      <Palette title={t('color.grays')} text={t('color.graysText')} colors={GRAY_PALETTE} />
+    </View>
+  )
+})
+
+const Backgrounds = withNamespaces(NameSpaces.brand)(
+  withScreenSize<I18nProps>(function _Backgrounds({ t, screen }: I18nProps & ScreenProps) {
+    const stylesForJudgemnt = screen === ScreenSizes.DESKTOP ? styles.column : [standardStyles.row]
+
+    return (
+      <View>
+        <SectionTitle containerStyle={brandStyles.gap}>{t('color.backgroundTitle')}</SectionTitle>
+        <Palette text={t('color.backgroundText')} colors={BACKGROUND_PALETTE} />
+        <Text style={[brandStyles.gap, fonts.h5a, standardStyles.elementalMarginBottom]}>
+          {t('color.contrast')}
+        </Text>
+        <Text style={[brandStyles.gap, fonts.p]}>{t('color.contrastText')}</Text>
+        <View style={[standardStyles.elementalMargin, standardStyles.row]}>
+          <Lorem
+            color={colors.dark}
+            backgroundColor={colors.white}
+            withGap={true}
+            hasBorder={true}
+          />
+          <Lorem color={colors.white} backgroundColor={colors.dark} withGap={true} />
+        </View>
+        <Text style={[brandStyles.gap, fonts.p]}>{t('color.contrastText2')}</Text>
+        <View style={screen === ScreenSizes.DESKTOP ? brandStyles.tiling : { width: '100%' }}>
+          <View style={stylesForJudgemnt}>
+            <Judgement is={Value.Bad}>
+              <Lorem color={colors.gold} backgroundColor={colors.faintGold} />
+            </Judgement>
+            <Judgement is={Value.Good}>
+              <Lorem color={colors.dark} backgroundColor={colors.faintGold} />
+            </Judgement>
+          </View>
+          <View style={stylesForJudgemnt}>
+            <Judgement is={Value.Bad}>
+              <Lorem
+                color={colors.white}
+                backgroundColor={colors.gray}
+                image={require('src/brandkit/images/lilah.jpg')}
+              />
+            </Judgement>
+            <Judgement is={Value.Good}>
+              <Lorem
+                color={colors.white}
+                backgroundColor={colors.dark}
+                image={require('src/brandkit/images/lilahOverlay.jpg')}
+              />
+            </Judgement>
+          </View>
+          <View style={stylesForJudgemnt}>
+            <Judgement is={Value.Bad}>
+              <Lorem color={colors.primary} backgroundColor={colors.deepBlue} />
+            </Judgement>
+            <Judgement is={Value.Good}>
+              <Lorem color={colors.white} backgroundColor={colors.deepBlue} />
+            </Judgement>
+          </View>
+        </View>
+      </View>
+    )
+  })
+)
+
+interface LoremProps {
+  backgroundColor: colors
+  color: colors
+  withGap?: boolean
+  hasBorder?: boolean
+  image?: ImageRequireSource
+}
+
+function Lorem({ backgroundColor, color, withGap, hasBorder, image }: LoremProps) {
+  const Component = image ? ImageBackground : View
+
+  return (
+    // @ts-ignore
```

Comment:
What's the error here?

Reply:
Component does not have call signature ?